### PR TITLE
Plugins print flags

### PR DESCRIPTION
**Pull request by skvl**

### CODE REVIEW

**Comment by skvl** [comment]
@tklengyel I will re-base the branch as soon as previous branch would be accepted.

**Comment by tklengyel** [comment]
@drakvuf-jenkins This is OK to test

File: src/plugins/plugin_utils.h [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+#include <map>
+#include <string>
+
+#include "plugins.h"
+
+using flags_str_t = std::map<uint64_t, std::string>;
+
+#define REGISTER_FLAG(FLAG) { FLAG, #FLAG }
+
+std::string parse_flags(uint64_t flags, flags_str_t flags_map, output_format_t format = OUTPUT_DEFAULT, std::string empty = std::string());
```
Comment (tklengyel) [inline src/plugins/plugin_utils.h]: Please add a comment that describes what this function does

**Comment by tklengyel** [comment]
@drakvuf-jenkins Retest this please

**Pull request merged**